app/commands/config.go

```diff
@@ -301,7 +302,7 @@ func setupLingo(c *cli.Context) (string, error) {
 	// TODO (Junyu) set proper Perforce Username and password
 	// Prompt for user
 	if username == "" || password == "" {
-		lingoTokenAddr := "http://" + webAddr + "/lingo-token"
+		lingoTokenAddr := "http://" + webAddr + "/profile/new-token"
```

Comment:
While you're in this area, might be good to remove the port from the address. I think now it will show `http://codelingo.io:80/profile/new-token`. Works all the same, just looks a bit ugly and redundant.